airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalAttemptExecution.java

```diff
@@ -56,7 +57,7 @@ public TemporalAttemptExecution(final Path workspaceRoot,
                                   final CheckedSupplier<Worker<INPUT, OUTPUT>, Exception> workerSupplier,
                                   final Supplier<INPUT> inputSupplier,
                                   final CancellationHandler cancellationHandler,
-                                  final JobPersistence jobPersistence,
+                                  final AirbyteApiClient airbyteApiClient,
```

Comment:
@cgardens @jdpgrailsdev @gosusnp discussion from parent PR:
![image](https://user-images.githubusercontent.com/7798112/186286089-1cc183a7-8e56-455a-89cb-c03de73dad38.png)

Reply:
I'll add a TODO comment to the WorkerApp PR that captures this